ArduCopter/ArduCopter.pde

```diff
@@ -743,7 +744,7 @@ static const AP_Scheduler::Task scheduler_tasks[] PROGMEM = {
     { ekf_check,            40,      2 },
     { crash_check,          40,      2 },
     { landinggear_update,   40,      1 },
-    { gcs_check_input,	     8,    550 },
+    { gcs_check_input,	     1,    550 },
```

Comment:
this is the most concerning change for the flight code. But it's required to ensure that the rate loop on the gimbal run's at 100Hz without too much latency between report and reply msgs form the gimbal